src/pixel_traits/het_pixel.rs

```diff
@@ -409,9 +409,39 @@ with_alpha!(GrayAlpha_v08, 2, [0], 1);
 
 without_alpha!(Bgr, 3, [b, g, r]);
 without_alpha!(Rgb, 3, [r, g, b]);
-without_alpha!(Grb, 3, [r, g, b]);
+without_alpha!(Grb, 3, [g, r, b]);
 without_alpha!(Gray_v09, 1, [v]);
 without_alpha!(Rgbw, 4, [r, g, b, w]);
 
 use crate::formats::gray::Gray_v08;
 without_alpha!(Gray_v08, 1, [0]);
+
```

Comment:
I think the unit tests are unnecessary for this personally, it's unlikely that this happens again. It was probably a copy-paste mistake.